page_data.go

```diff
@@ -4,90 +4,120 @@ import "time"
 
 const pageDataURL = "/w/api.php"
 
+// PageDataOresScore representation for ORES score
+type PageDataOresScore struct {
+	True  float64 `json:"true"`
+	False float64 `json:"false"`
+}
+
+// PageDataOresArticleQuality representation for ORES article quality
+type PageDataOresArticleQuality struct {
+	Stub float64 `json:"Stub"`
```

Comment:
I think json tag should be `stub`

Reply:
The `Stub` is correct, for some reason actions API returns it that way.